skops/card/_model_card.py

```diff
@@ -402,6 +402,15 @@ def add_permutation_importances(
         self : object
             Card object.
         """
+        try:
+            metadata.version("matplotlib")
```

Comment:
instead, just try to import here rather than getting the version, and remove the import a few lines after.